scl/Makefile.am

```diff
@@ -25,6 +25,7 @@ SCL_SUBDIRS	= \
 	sudo		\
 	syslogconf	\
 	system		\
+	websense	\
```

Comment:
Please add this to scl/CMakeLists.txt too.

Reply:
done